docker/mysql.repo

```diff
@@ -0,0 +1,5 @@
+[mysql56-community]
+name=MySQL 5.6 Community Server
+baseurl=http://repo.mysql.com/yum/mysql-5.6-community/el/7/$basearch/
+enabled=1
+gpgcheck=0
```

Comment:
Would be nice to import the necessary keys here so we don't have to turn this off.